docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -77,10 +100,11 @@ To see more details about each related process, select the process either in the
 [role="screenshot"]
 image::images/process-details.png[]
 
-See event details by selecting the **# events** URL at the top of the process details view or choosing one of the event pills in the graphical view.
+Loading process appear in a loading state when selecting that process for the first time. If loading data for a given process fails, simply click **Reload {process-name}** underneath the process to reload the data.
 
+See event details by selecting the **# events** URL at the top of the process details view or choosing one of the event pills in the graphical view.
```

Comment:
- `**# events** URL` - is this intentional? 
- "choosing one of the event pills" - I believe these are child processes (processes that spawned from the main process) - @jonathan-buttner or @kqualters-elastic or should we call these "events"? I want to make sure we get the terminology correct - thanks!

Reply:
When we say _**event pills**_ we're talking about the rectangles in the image below (e.g. 491 authentication). I think I'm fine with _**event pills**_ or _**process pills**_, the pills represent event categories but you could also say a particular process (node in the graph) has pills.

![image](https://user-images.githubusercontent.com/56361221/106057048-697fca00-60bd-11eb-966a-417c63bfa422.png)